app/models/forem_instance.rb

```diff
@@ -13,6 +13,23 @@ def self.email
     ApplicationConfig["DEFAULT_EMAIL"]
   end
 
+  def self.reply_to_email_address
+    # Some business logic context:
```

Comment:
I added this long-ish comment here because it takes some time to wrap ones head around how the SMTP settings are affected by self host vs Forem Cloud.